src/actions/admin.js

```diff
@@ -461,6 +461,30 @@ export function uploadPlugin(fileData: File, force: boolean = false): ActionFunc
     };
 }
 
+export function installPluginFromUrl(url: string, force: boolean = false): ActionFunc {
+    return async (dispatch, getState) => {
+        dispatch({type: AdminTypes.INSTALL_PLUGIN_FROM_URL_REQUEST, data: null});
+
+        let data;
+        try {
+            data = await Client4.installPluginFromUrl(url, force);
+        } catch (error) {
+            forceLogoutIfNecessary(error, dispatch, getState);
+            dispatch(batchActions([
```

Comment:
Do we need to send this as a batch action (if it's just one action)?

Reply:
We sure don't :wink:

Good catch!

Reply:
@cpoile Actually this one is dispatching two actions, but the one your comment below references should be fixed.

Reply:
Also, it seems other actions in this file are doing the same unnecessary `batchActions()` call, not that it matters for the scope of this PR.